recipes/euslisp-mode

```diff
@@ -1 +1,4 @@
-(euslisp-mode :repo "iory/euslisp-mode" :fetcher github)
+(euslisp-mode :fetcher github
+              :repo "iory/euslisp-mode"
+              :files ("euslisp-mode.el" "euslisp-find-defnition.el" "find_euslispfunction.py")
```

Comment:
You should use `:defaults` instead of specifying each `*.el` files(See https://github.com/melpa/melpa#recipe-format). (BTW you typo `euslisp-find-defnition.el`. `defnition` -> `definition`)